### PR TITLE
63   implement event element matrix

### LINKED ISSUE #63: Implement Event-Element matrix

### PR DESCRIPTION
**Pull request by Exclaminator**
Will close #63

Reopening of #78

### CODE REVIEW

File: resources/options.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     <script src="jquery.min.js"></script>
     <script src="materialize.min.js"></script>
     <script src="options.js"></script>
+    <!--TODO: add the do not watch options -->
```
Comment (thervh70) [inline resources/options.html]: Could you explain why this has to be done?
I don't understand why you need the TypeScript file here.

Comment (Exclaminator) [inline resources/options.html]: What typescript file are you talking about? this is just a memo that UI should be made for these new options.

Comment (thervh70) [inline resources/options.html]: Oh wait, I'm sorry, I had to continue reading. I understand what you have done.
My bad.

**Comment by thervh70** [comment]
Like the new changes!

Ready for merge.

**Comment by Exclaminator** [comment]
Cool, @mpsijm also already agreed (prev branch, and I discussed with him in the lecture) so I will merge this request.

**Pull request merged**